app/services/taxonomy/build_taxon_payload.rb

```diff
@@ -4,32 +4,35 @@ def initialize(taxon)
       @taxon = taxon
     end
 
-    def self.call(taxon:)
-      new(taxon).build
+    def self.call(taxon:, locale: "en")
+      new(taxon).build(locale)
     end
 
-    def build
-      {
-        base_path: base_path,
+    def build(locale)
+      path = locale == "en" ? base_path : base_path + ".#{locale}"
```

Comment:
Minor thing I didn't spot before: it might be clearer to make this a memoised private method.

Reply:
Done 👍